src/sagemaker/image_uri_config/pytorch.json

```diff
@@ -263,6 +264,7 @@
             },
             "1.4.0": {
                 "py_versions": [
+                    "py2",
```

Comment:
Can we try removing this since now that the inference lower bound is corrected.
Sorry for the mistake